docs/installing-solidity.rst

```diff
@@ -60,17 +60,33 @@ Please refer to the solc-js repository for instructions.
 Docker
 ======
 
-We provide up to date docker builds for the compiler. The ``stable``
-repository contains released versions while the ``nightly``
-repository contains potentially unstable changes in the develop branch.
+Docker images of Solidity builds are available. Use the ``stable`` tag for the
+latest released version, and ``nightly`` for potentially unstable changes in the develop branch.
+
+.. code-block:: bash
+
+    docker run ethereum/solc:stable
+
+You can also specify a release build version in the tag, for example, for the 0.5.4 release.
+
+.. code-block:: bash
+
+    docker run ethereum/solc:0.5.4
+
+The Docker image contains the compiler executable, so you can pass all compiler arguments to it. For example.
 
 .. code-block:: bash
 
     docker run ethereum/solc:stable --version
 
-Currently, the docker image only contains the compiler executable,
-so you have to do some additional work to link in the source and
-output directories.
+To use the Docker image to compile Solidity files on the host machine mount a
+local folder for input and output, and specify the contract to compile. For example.
+
+.. code-block:: bash
+
+    docker run -v /local/path:/mounted/path ethereum/solc:stable -o /local/path/output --abi --bin /local/path/contract.sol
```

Comment:
Shouldn't it be
```suggestion
    docker run -v /local/path:/mounted/path ethereum/solc:stable -o /local/path/output --abi --bin /mounted/path/contract.sol
```
instead? Also maybe just use `/sources` in the docker image, since that actually does not have to be specific to the user.

Reply:
Ahh, you're right, it's local, but the mounted path.